ldaptor/delta.py

```diff
@@ -129,7 +129,7 @@ def __init__(self, dn, modifications=[]):
 
     def asLDIF(self):
         r = []
-        r.append(ldif.attributeAsLDIF('dn', str(self.dn)))
+        r.append(ldif.attributeAsLDIF('dn', self.dn.toWire().decode('utf-8')))
```

Comment:
This is fine... the internal decoding is a bit strange... but this is a direct translation.

I was expecting for `ldif` to have a method which can taked a name and  DN and add it directly without the need of serialization and deserialization... something like


```suggestion
        r.append(ldif.add('dn', self.dn))
```

Reply:
True. Appropriate changes were made in ``ldif`` module functions, they now convert passed arguments to their bytes representation using ``to_bytes``.